code/game/machinery/computer/arcade/arcade.dm

```diff
@@ -285,6 +285,12 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	if(usr?.mind)
 		gamerSkill = usr.mind.get_skill_level(/datum/skill/gaming)
 
+	SEND_SIGNAL(usr, COMSIG_ADD_MOOD_EVENT, "gaming", /datum/mood_event/gaming)
+	// Quell gaming need for gamers
+	var/datum/quirk/gamer/gamer_quirk = locate() in usr.mind.current.quirks
+	if(gamer_quirk)
+		gamer_quirk.gamed()
```

Comment:
why do this instead of just registering the signal you are adding, to the quirk?

Reply:
I'm not quite sure what you mean, sorry. Do you mean register the `gamed`, `lost_game`, and `won_game` procs to a signal, and execute them via that, rather than doing this whole locate quirk thing?

Reply:
yes